com.unity.render-pipelines.high-definition/Editor/Unity.RenderPipelines.HighDefinition.Editor.asmdef

```diff
@@ -30,6 +30,11 @@
             "name": "com.unity.xr.management",
             "expression": "1.0.0",
             "define": "ENABLE_XR_MANAGEMENT"
+        },
+        {
+            "name": "com.unity.rendering.hybrid",
+            "expression": "0.14.0-preview.0",
+            "define": "ENABLE_HYBRID_RENDERER_V2"
         }
```

Comment:
It impacts the usage of 
- [DecalSubTarget](https://github.com/Unity-Technologies/Graphics/blob/10.x.x/release/com.unity.render-pipelines.high-definition/Editor/Material/Decal/ShaderGraph/DecalSubTarget.cs#L491)
- [HDTarget](https://github.com/Unity-Technologies/Graphics/blob/10.x.x/release/com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDTarget.cs#L717)
It looks good to me, also discussed about this with @alelievr